### PR TITLE
Chore: update the readme file

### PR DESCRIPTION
**Pull request by andresgalante**
> [Download the latest build](https://github.com/blockstack/stacks-wallet/actions/runs/625046871)

This PR tries to improve a bit the installing instructions and adds a link to our test coverage.

@kyranjamie can you please review if this is written in the queen's English?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
-Implementation of the Stacks 2.0 wallet
+Implementation of the Stacks 2.0 wallet for Desktop
 
 ## Getting started
 
-```sh
-yarn
-```
+1. Ensure you have [Yarn](https://yarnpkg.com/) installed.
+1. After [cloning the repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository), run `yarn` in the root of the project to install the dependencies.
```
Comment (kyranjamie) [inline README.md:12]:
```suggestion
2. After [cloning the repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository), run `yarn` in the root of the project to install the dependencies.
```

File: README.md [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+1. Run `yarn dev:testnet` to start a development server.
```
Comment (kyranjamie) [inline README.md:13]:
```suggestion
3. Run `yarn dev:testnet` to start a development server.
```

**Comment by kyranjamie** [comment]
Thanks @andresgalante, don't forget to set your git username & email https://linuxize.com/post/how-to-configure-git-username-and-email/

File: README.md (hunk 2 above)
Comment (andresgalante) [reply README.md:13]: I don't think we need to number them, Markdown takes `1.` as an `ol`

**Review by kyranjamie — APPROVED** [review]

**Comment by andresgalante** [comment]
Is this ok to merge? who merges it?

**Pull request merged**